CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Serve static files from `blobs` directory [#480](https://github.com/p2panda/aquadoggo/pull/480)
 - Add method to store for pruning document views [#491](https://github.com/p2panda/aquadoggo/pull/491)
 - Introduce `BlobStore` [#484](https://github.com/p2panda/aquadoggo/pull/484)
+- `garbage_collection` materialization task [#500](https://github.com/p2panda/aquadoggo/pull/500)
```

Comment:
Think its nice to explain a bit more _what_ got added here

Reply:
.. the name of this PR is already much better: "Task for automatic garbage collection of unused documents and views"